aws/chef-bootstrap.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+# Code-dot-org Chef bootstrap script.
+# One-liner to run:
+# aws s3 cp s3://cdo-dist/chef/bootstrap.sh - | sudo bash -s -- [options]
```

Comment:
This comment is a little confusing since it runs a static snapshot in s3 not the version the user is reading.

What version of this file do we expect to find in s3?